sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -452,14 +453,20 @@ export class EventHubReceiver extends LinkEntity {
 
         if (!this.isOpen()) {
           try {
-            await this.initialize();
-            if (abortSignal && abortSignal.aborted) {
-              await this.abort();
-            }
+            await this.initialize({
+              abortSignal,
+              timeoutInMs: getRetryAttemptTimeoutInMs(this.options.retryOptions)
+            });
           } catch (err) {
             if (this._onError === onError) {
               onError(err);
             }
+            if (err.name === "AbortError") {
```

Comment:
We had code in the error callback to registerHandlers to handle AbortError, so why do we need to do this?

Reply:
You're right, we don't. Removing.